### ISSUE TITLE
Crash on doileak.com

### ISSUE THREAD

**Issue by avently** [brave/browser-android-tabs#298]
When I'm clicking Start test here -> https://doileak.com <- I'm getting crash..

Here is the log:

07-10 14:14:30.594 F/chromium(10476): [FATAL:dom_storage_area.cc(312)] Check failed: !session_storage_backing_.get(). 

Can you take a look?

**Comment by SergeyZhukovsky** [comment]
@avently it is an assert. It fires on debug mode only. Developers use them to point that something is going wrong and that it is something that worth to look at in the future. However it is not critical or just not enough clear how to fix it and it doesn't crash on release mode.

**Comment by avently** [comment]
@SergeyZhukovsky thank you. I will rebuild in release mode. Do you know russian language?

**Comment by SergeyZhukovsky** [comment]
Great! Yes I do.

**Comment by avently** [comment]
@SergeyZhukovsky а в Google Play загружена программа, собранная в debug mode? Потому что и она крэшится на doileak.com после нажатия кнопки Start test.

**Comment by SergeyZhukovsky** [comment]
@avently let's keep a conversation on English. We are open source and lots of people see our comments. It would be nice if most people can understand what we are talking about. 
I verified it and you are right the release version(Google Play contains the release version) crashes as well. I will reveal that ticket as it is something to investigate. However I tried it on Chrome as well which is based on Chromium and it crashes on that page also. And I have a crash in different place

07-10 21:46:12.032: A/DEBUG(11350): Abort message: '[FATAL:DocumentLifecycle.cpp(350)] Check failed: CanRewindTo(state). Cannot rewind document lifecycle from "kInPaint" to "kVisualUpdatePending".
